src/main/java/com/codeborne/selenide/conditions/MatchAttributeWithValue.java

```diff
@@ -0,0 +1,40 @@
+package com.codeborne.selenide.conditions;
+
+import com.codeborne.selenide.Condition;
+import com.codeborne.selenide.Driver;
+import org.openqa.selenium.WebElement;
+
+import java.util.regex.Pattern;
+
+import static java.util.regex.Pattern.DOTALL;
+
+public class MatchAttributeWithValue extends Condition {
+  private final String attributeName;
+  private final Pattern attributeRegex;
+
+  public MatchAttributeWithValue(String attributeName, String attributeRegex) {
+    super("match attribute");
+    this.attributeName = attributeName;
+    this.attributeRegex = Pattern.compile(attributeRegex, DOTALL);
```

Comment:
Why DOTALL?
As I know, it's needed to support multiline Strings, which hardly can happen to attribute values.

Reply:
Thanks for review Andrei! 
You are right! I removed the flag, seems this is not needed there (at least, I did not find appropriate)